test-requirements.txt

```diff
@@ -1,4 +1,3 @@
-codecov
 coverage
```

Comment:
do we still need the 'coverage' module?

Reply:
As per my understanding we need it for this but @maddieford can test it without the module.

echo looking for coverage files :
        ls -alh | grep -i coverage
        sudo env "PATH=$PATH" coverage combine coverage.*.data
        sudo env "PATH=$PATH" coverage xml
        sudo env "PATH=$PATH" coverage report

Reply:
I see, so "codecov/codecov-action@v2" only uploads the report, but we still need to produce it. All good then.